### ISSUE TITLE
Identify for AGS layers doesn't work

### ISSUE THREAD

**Issue by brentfraser** [geomoose/gm3#512]
GM 3.6
Turn on the "AGS Dakota County Rail" layer and do an identify on one of the lines.
No results are shown.

The network traffic is

```
https://gis2.co.dakota.mn.us/arcgis/rest/services/DCGIS_OL_Transportation/MapServer/16/query/
?f=json
returnGeometry=true
spatialReference={"wkid":102100}
inSR=102100
outSR=102100
outFields=*
layers=[{"layerId":"railroads","where":""}]
callback=reqwest_1590680382445
```
Note the lack of 

`geometry=-10389315.44,5584710.51`

in the request.

The server returns:
`
reqwest_1590680382445({"error":{"code":400,"message":"","details":[]}});`

which GM doesn't handle very well #505, #392

**Comment by brentfraser** [comment]
Adding the "geometry=-10389315.44,5584710.51" to the request makes it successful.

**Comment by brentfraser** [comment]
looking at map.js, lines 473, 475, 486 need to be changed to use
`query.selection[0].geometry`
instead of 
`query.selection.geometry`

Making these changes no generates an error, but the server returns no features.

**Comment by brentfraser** [comment]
fixed in #513
